docs/ui/emacs.rst

```diff
@@ -163,6 +165,9 @@ and show the goals at the current line with ``C-C C-c``. As always with
 emacs, if you were to be dissatisfied with these keybindings, you can
 change them easily!
 
+To focus a goal, you can click on the goal in the ``*Goals*`` buffer
```

Comment:
Sorry, I'm a bit picky, but could you precise what you mean by "focus"? Because initially I thought it had to do with "window focusing". It may be
> Clicking on a goal in the ``*Goals*`` buffer focuses it in the proof script by inserting a ``focus G`` tactic

but if you have a better formulation, be my guest!

Reply:
I agree, that could lead to some confusion